test/e2e/create_test.go

```diff
@@ -681,13 +681,18 @@ var _ = Describe("Podman create", func() {
 		create := podmanTest.Podman([]string{"create", "--uidmap", "0:1000:1000", "--pod", "new:testing123", ALPINE})
 		create.WaitWithDefaultTimeout()
 		Expect(create).ShouldNot(ExitCleanly())
-		Expect(create.ErrorToString()).To(ContainSubstring("cannot specify a new uid/gid map when entering a pod with an infra container"))
+		Expect(create.ErrorToString()).To(ContainSubstring("cannot set user namespace mappings that differ from pod"))
+
+		cleanup := podmanTest.Podman([]string{"pod", "rm", "-f", "testing123"})
+		cleanup.WaitWithDefaultTimeout()
```

Comment:
this is missing a `Expect(session).Should(Exit(0))` to make sure the command worked.

New test code however should just use PodmanExitCleanly which combines this all into a one liner
```
podmanTest.PodmanExitCleanly("pod", "rm", "-f", "testing123")
```